config.js

```diff
@@ -30,6 +30,14 @@ var config = { // eslint-disable-line no-unused-vars
     // Required version of Chrome extension
     desktopSharingChromeMinExtVersion: '0.1',
 
+    // The desired resolution of the desktop preview images to use for
```

Comment:
Hum, come to thing of it, I'm not sure how useful this can be :-S Sorry for the noise, we should probably take that back. I thought we used it in a few places, but I see we don't. Sorry!